### PR TITLE
New functionality for DSC Extension - Publish-AzureVMDscConfiguration cmdlet(have merge conflicts)

### PR DESCRIPTION
**Pull request by eshaparmar**
*We have added new parameters/functionality for the DSC Extension - Publish-AzureVMDscConfiguration cmdlet (both arm/asm). 

*Tests for these are written in powershell in our code base and run through our daily CI system

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@eshaparmar__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (esparmar). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by eshaparmar** [comment]
@Azure/adx-sdk-team

**Comment by stankovski** [comment]
@azuresdkci retest this please

**Comment by yugangw-msft** [comment]
please resolve the confliction

**Comment by stankovski** [comment]
Started on-demand build: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-on-demand/42/

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/UnitTests/Cmdlets/IaaS/Extensions/DSC/GetAzureVMDscExtensionStatusUnitTest.cs [hunk 1]
```diff
@@ -0,0 +1,273 @@
+﻿using System;
```
Comment (yugangw-msft) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/UnitTests/Cmdlets/IaaS/Extensions/DSC/GetAzureVMDscExtensionStatusUnitTest.cs]: license header

**Comment by yugangw-msft** [comment]
your post check in test failed
http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-on-demand/42/artifact/src/Publish/TestResults/Microsoft.WindowsAzure.Commands.Common.Test.html

**Comment by eshaparmar** [comment]
I talked to Denis offline about it. Those tests are working locally but for some reason they fail now on build system. I have disabled those tests temporarily. Please merge this PR

**Comment by eshaparmar** [comment]
Closing this PR and created a new one #689

**Pull request closed without merging**